src/uct/ib/base/ib_iface.c

```diff
@@ -236,13 +239,30 @@ ucs_status_t uct_ib_iface_recv_mpool_init(uct_ib_iface_t *iface,
                         config->rx.mp.max_bufs);
     }
 
+    status = uct_iface_param_am_alignment(params, UCS_SYS_CACHE_LINE_SIZE, 0,
```

Comment:
seems we always pass (UCS_SYS_CACHE_LINE_SIZE, 0) to this function
maybe remove those args?